Automattic/wp-calypso
# Jetpack Connect: Add "from" to tracks analytics

This PR adds `from` to tracks analytics.

Closes #21599 

## Testing
1. Run in the browser console:
   ```js
   localStorage.debug = 'calypso:analytics:tracks'
   ```
1. Connect a new site via `/jetpack/connect`
1. See the modified analytics calls including `from`, for example `calypso_jpc_authorize_form_view`
1. View the live tracks events on the dashboard and verify `from` is included and reflects the changes here.
